src/components/Anniversary/Slider/styles.postcss

```diff
@@ -76,10 +76,11 @@
     }
 
     .slider-element {
+      position: relative;
       display: flex;
       flex-flow: column;
       justify-content: center;
-      background-color: $orange-tangerine;
+      background-color: $orange-yellow;
```

Comment:
Can you use these colors as $color-n ?
This is to avoid having to look for all the recurrences in your code if you need to change them.